rspack_hmr/app2/rspack.config.js

```diff
@@ -0,0 +1,122 @@
+const rspack = require("@rspack/core");
+const refreshPlugin = require("@rspack/plugin-react-refresh");
+const isDev = process.env.NODE_ENV === "development";
+
+const path = require("path");
+const deps = require("./package.json").dependencies;
+console.log({ deps });
+const { ModuleFederationPlugin } = require("@module-federation/enhanced/rspack");
+
+const name = "app_02";
```

Comment:
const fileName = ...
const uniqueName = fileName + "1"

As a suggestion. Do you need them to be different?

Reply:
i thiiiink that it needs a unique name for hmr to work properly. +1 isn't ideal, but simple for a sample project